src/app/recommendations/constants.ts

```diff
@@ -6,7 +6,7 @@ export const COLUMN_DATA: [string, string, boolean][] = [
   ["TIS Status", "doctorStatus", false],
   ["Programme name", "programmeName", false],
   ["Programme membership type", "programmeMembershipType", false],
-  ["CCT date", "cctDate", false],
+  ["Curriculum end date", "cctDate", false],
```

Comment:
Can we change `cctDate`? Continuing with that name is likely to confuse "future Pepe".

Reply:
Thanks @JosephKelly, it's a good shout. Was thinking about it. In order to make it consistent, we may need to update all names in TCS, Recommendation, Concerns, Core, Integration as well. 
Will make PRs for the backend shortly. Then we merge the frontend as the last one.